elements/pfe-progress-steps/src/pfe-progress-steps.scss

```diff
@@ -0,0 +1,81 @@
+@import "_shared-assets";
+
+$LOCAL: progress-steps;
+
+$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
+  progress-bar: (
+    color: pfe-var(ui--border--lighter),
+    color--active: pfe-var(feedback--info),
+    thickness: pfe-var(ui--border-width--md)
+  ),
+));
+
+:host {
+  display: block;
+  display: flex;
+  width: auto;
+  justify-content: space-between;
+  position: relative;
+  align-items: flex-start;
+  
+  // @TODO This definitely would need some IE love since it's driven by CSS variables so I'm just not displaying it in IE for now.
+  @include browser-query(ie11) {
+    display: none;
+  }
+}
+
+// Make everything vertical
+:host([vertical]) {
+  flex-direction: column;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+.pfe-progress-steps__progress-bar {
+  display: flex;
+  position: absolute;
+
+  top: calc(
+    var(--pfe-progress-steps-item__circle--size--active, #{pfe-fetch(size--active, $region: circle)}) -
```

Comment:
```suggestion
  top: calc(
    #{pfe-local(size--active, $region: circle)} -
    #{pfe-local(size, $region: circle)} +
    #{pfe-local(thickness, $region: progress-bar)}
  );
```



```css
.before {
  top: calc( 32px - 20px + 2px);
  top: calc( var(--pfe-progress-steps-item__circle--size--active, var(--pfe-theme--ui--element--size--md, 32px)) - var(--pfe-progress-steps-item__circle--size, var(--pfe-theme--ui--element--size, 20px)) + var(--pfe-progress-steps__progress-bar--thickness, var(--pfe-theme--ui--border-width--md, 2px)));
}

.after {
  top: calc( 32px - 20px + 2px);
  top: calc( var(--pfe-progress-steps__circle--size--active, var(--pfe-theme--ui--element--size--md, 32px)) - var(--pfe-progress-steps__circle--size, var(--pfe-theme--ui--element--size, 20px)) + var(--pfe-progress-steps__progress-bar--thickness, var(--pfe-theme--ui--border-width--md, 2px)));
}
```
I think the pfe-local function generates "more better" variable names. This variable `--pfe-progress-steps-item__circle--size` is not BEM'y because I'm unsure which thing we're talking about, the item? or the circle? If I reference the local variables map, I think we are just talking about the circle, so the var should be `--pfe-progress-steps__circle--size`.

Reply:
This component is comprised of `pfe-progress-steps` with nested `pfe-progress-steps-item`.  This example is passing the value of the circle size down to each of the `progress-steps-item`s.

This variable: `--pfe-progress-steps-item__circle--size` equates to the circle region of the `progress-steps-item` and is providing the size value.  In this way it is _very_ BEM.

This is why the pfe-local function cannot be used here.  The parent component (the file this is in) `pfe-progress-steps` is the value of the $LOCAL variable; that means, without changing the value of `$LOCAL`, pfe-local cannot be used to capture the value for the child elements.